src/components/home/partners/SecondaryPartners.jsx

```diff
@@ -65,25 +67,29 @@ const LargeScreenComponent = ({
             alt={`${company} logo`}
             aria-label={`${company} logo`}
             src={partnerLogo}
-            mb={'150px'}
             width={'250px'}
           ></Box>
         </a>
       </Grid>
-      <Grid item pl={'6%'} {...partnerGridStyle} order={contentOrder}>
-        <Typography variant="body1" p={'5% 15% 3% 0'}>
+      <Grid item pl={'6%'} pr={'1%'} {...partnerGridStyle} order={contentOrder}>
+        <Typography variant="body1" p={'5% 0 3% 0'}>
           {testimonial}
         </Typography>
-        <Typography variant="caption" p={'15px 0 5% 0'}>
+        <Typography variant="caption" sx={{ fontWeight: 'bold' }}>
           {testimonialAuthor}
         </Typography>
         {testimonialTwo ? (
```

Comment:
on both `LargeScreenComponent` and `SmallScreenComponent` there are ternaries used that are just something to the effect of:

```js
{condition ? (
    <Returned JSX>
) : null}
```

if it's "this" or "null", I think it's more commonplace in React (and I'd personally prefer) to use the JS AND operand short-circuiting instead:

```js
{condition && (
    <Returned JSX>
)}
```

if condition is true, the right of AND gets evaluated and rendered.